docs/operations/annotations_en.md

```diff
@@ -0,0 +1,96 @@
+# Annotations for the k8s-dogu-operator
+
+This document contains the detailed description of all generated annotation of the k8s-dogu-operator.
+
+## Services
+
+This section contains all annotation that are appended to K8s services, if necessary.
+
+### k8s-dogu-operator.cloudogu.com/ces-services
+
+The annotation `k8s-dogu-operator.cloudogu.com/ces-services` contains information about one or multiple ces services.
+Each ces service defines an external service of the system reachable via the web server. The annotation is automatically
+generated for every dogu which is marked as webapp. It is also possible to adapt the behaviour of the services, i.e., providing
+custom url by which the service is available. 
```

Comment:
```suggestion
a custom URL by which the service is available. 
```

Reply:
The English document is automatically generated.